tests/test_generic/tests/test_xml.py

```diff
@@ -34,7 +34,7 @@ def _check_files_in_path(self, module):
             for file_name in files:
                 file_path = os.path.join(root, file_name)
                 ext = os.path.splitext(file_path)[1].lower()
-                if 'static/' in file_path and 'description' not in file_path:
+                if not file_path.endswith('.js') and ('static/' in file_path and 'description' not in file_path):
```

Comment:
Couldn't we also check that the tour exists?

Reply:
What do you mean the tour exists?

Reply:
I'd add a few tests to make sure we don't forget all these must-have
- id welcome_article
- js tour in test_generic
- check the mail message 
- ...